packages/app/pages/schedule.tsx

```diff
@@ -0,0 +1,33 @@
+import { Calendar, momentLocalizer } from "react-big-calendar";
+import moment from "moment";
+import useSWR from "swr";
+import { API } from "@template/api-client";
+import { Result } from "antd";
+
+export default function Schedule() {
+  const { data, error } = useSWR(`api/v1/courses/1`, async () =>
```

Comment:
Might be slightly easier to just use the `/v1/courses/{course_id}/` for the routes so we don't have to add the extra `/api/` onto all of them but that's not a big deal.

Reply:
we have to prepend everything with `api` due to some gobblygook